docs/guides/transactions.md

```diff
@@ -6,6 +6,11 @@ A transaction is basically a context manager that collects all file uploads, def
 They are an "all or nothing" proposition: If an error occurs during the transaction, none of the queued files are uploaded.
 For more information on fsspec transactions, see the official [documentation](https://filesystem-spec.readthedocs.io/en/latest/features.html#transactions).
 
+!!! info
```

Comment:
The placement of this callout is a bit (visually) unfortunate, IMO, since it separates the last sentence of the introduction from the rest, creating a sort-of [orphan](https://en.wikipedia.org/wiki/Widows_and_orphans).

Instead of a list inside the callout, I would simply have two sentences (maybe with `* *` emphasis on the package names).

In case you want to keep the list, please include a blank link before the first list item, otherwise the list does not render correctly in Mkdocs.